### PR TITLE
Added to LabyMod disclaimer, made note about mutes

### PR DESCRIPTION
**Pull request by calcastor**
hoping I didn't somehow screw up the code...

### CODE REVIEW

File: _pages/rules.html [hunk 1]
```diff
@@ -64,7 +64,7 @@ <h2 class="page-heading">C. Client Mods</h2>
           </ul>
           <li>For any other mods, email <a href="mailto:brott@oc.tc">brott@oc.tc</a> for confirmation before using it.</li>
           <span class="badge badge-warning">Note</span>
-          <small>Unless explicitly permitted, additional mods which can be installed through LabyMod are not allowed.</small>
+          <small>Unless explicitly permitted, additional mods which can be installed through LabyMod that affect gameplay are not allowed. When installing LabyMod, you must not install its Damage Indicator mod.</small>
```
Comment (Brottweiler) [inline _pages/rules.html:67]: I find this redundant. Does Labymod have damage indicators by default?

File: _pages/rules.html [hunk 2]
```diff
@@ -94,6 +94,7 @@ <h2 class="page-heading">E. Punishments</h2>
     <div class="col-md-12">
       <ol>
         <li>Players may be kicked for breaking the rules. If the rule violation is severe, you may be banned instead.</li>
+        <li>Disrespectful and/or toxic behaviour in chat may result in a mute. The mute length can vary depending on previous offences.</li>
```
Comment (Brottweiler) [inline _pages/rules.html:97]:
```suggestion
        <li>Players may be muted for breaking chat rules. If you have been muted before, the mute length may be longer.</li>
```
Something like this instead?

**Review by Brottweiler — CHANGES_REQUESTED** [review]

File: _pages/rules.html (hunk 1 above)
Comment (calcastor) [reply _pages/rules.html:67]: image

LabyMod has a few mods you can install out of the box, with several others available as downloadable mods in game. It's one of the mods that is available to add during install, but perhaps we do not need to add a note if we have the note above (I had misremembered it being selected by default).

File: _pages/rules.html (hunk 2 above)
Comment (calcastor) [reply _pages/rules.html:97]: Sounds good.

File: _pages/rules.html (hunk 1 above)
Comment (Brottweiler) [reply _pages/rules.html:67]: I think that is what is referred to with "additional mods which can be installed through LabyMod are not allowed".

Comment (calcastor) [reply _pages/rules.html:67]: My reasoning behind the further clarifications lies in the fact that the vast majority of LabyMod's optional mods are features that exist in Badlion Client and Lunar Client, which are clients that are explicitly permitted. For example, mods like ItemPhysics, Laby's Keystrokes, Resourcepacks24, More Particles etc. that are available to add when installing LabyMod itself are mostly cosmetic or for adding extra info on the HUD during play that exist in both/either Badlion and Lunar, while others that may not be permitted are only enabled on servers if the server explicitly seeks out to permit it, such as Laby's Minimap. Other mods which you can download within the client, such as Custom Time or Full Bright, or Better Perspective are all features integrated into Badlion and Lunar. 

This being said I don't think I'm happy anymore with how I've worded it in this PR. I do think how it's worded previously might be confusing for some, given that Badlion and Lunar are explicitly permitted with many of the same features integrated as opposed to available for download after installation.

**Pull request closed without merging**